File: generated-sources/scala-finch/mojang-status/src/main/scala/org/openapitools/models/ApiStatus.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._

/**
 * The status of a Mojang services. Possible values are green (no issues), yellow (some issues), red (service unavailable).
 */
case class ApiStatus()

object ApiStatus {
    /**
     * Creates the codec for converting ApiStatus from and to JSON.
     */
    implicit val decoder: Decoder[ApiStatus] = deriveDecoder
    implicit val encoder: ObjectEncoder[ApiStatus] = deriveEncoder
}

File: generated-sources/csharp-nancyfx/mojang-status/src/Org.OpenAPITools/Models/ApiStatus.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;
using Sharpility.Extensions;
using NodaTime;

namespace Org.OpenAPITools..Models
{
    /// <summary>
    /// The status of a Mojang services. Possible values are green (no issues), yellow (some issues), red (service unavailable).
    /// </summary>
    /// <value>The status of a Mojang services. Possible values are green (no issues), yellow (some issues), red (service unavailable).</value>
    public enum ApiStatus
    {
        
        /// <summary>
        /// Enum Green
        /// </summary>
        Green = 1,
        
        /// <summary>
        /// Enum Yellow
        /// </summary>
        Yellow = 2,
        
        /// <summary>
        /// Enum Red
        /// </summary>
        Red = 3
    }

}